Dockerfile

```diff
@@ -36,6 +34,12 @@ RUN useradd -m -s /bin/bash ${USERNAME} && \
     usermod -u ${UID} ${USERNAME} && \
     usermod -g ${GID} ${USERNAME}
 
+# NOTE(ycho): As the root user, mount host ssh and clone our packages into `tmp`.
+RUN --mount=type=ssh mkdir -p -m 0600 "/root/.ssh" \
+    && ssh-keyscan github.com | sudo tee -a "/root/.ssh/known_hosts"
+RUN --mount=type=ssh git clone git@github.com:iMSquared/imm-summit-packages.git --depth 1 -b full \
```

Comment:
`-b full` is temporary, we should probably update `imm-summit-packages` to be a working version since we made it private for now anyway, and refactor later as needed. (It turns out it's not actually that big)